trunk/NDHMS/hydro_namelists.json

```diff
@@ -3,32 +3,32 @@
         "hydro_nlist": {
             "aggfactrt": 4,
             "channel_option": 2,
-            "chanobs_domain": 0,
+            "chanobs_domain": 1,
             "chanrtswcrt": 1,
             "chrtout_domain": 1,
-            "chrtout_grid": 0,
+            "chrtout_grid": 1,
             "compound_channel": true,
             "dtrt_ch": 300,
             "dtrt_ter": 10,
             "dxrt": 250,
-            "frxst_pts_out": 0,
+            "frxst_pts_out": 1,
```

Comment:
Base is NWM-ish, correct?  If that's the case, I don't think we can even write forecast points output.  I could be wrong here though.